### DISCUSSION TITLE
Balance and reward doesn't render properly in new app

### DISCUSSION THREAD

**Discussion by danclee336** [clearpool-finance/community#8] [Bug reports]
I discovered that my balance and rewards render correctly on the old app: https://clearpool.finance/permissionless/positions?market=ethereum, but not on the new app: https://clearpool.finance/permissionless/positions?market=ethereum. The numbers on the new app are all 0 when it says otherwise in the old app.

Steps to reproduce: View Pool -> My Positions.

**Comment by zolotokrylin** [comment]
Thanks for the report, @danclee336. We will check and revert soon.

**Comment by zolotokrylin** [comment]
The issue was fixed.
